pages/common/views/pdf/header.jsx

```diff
@@ -1,43 +1,55 @@
   <Wrapper name="header" nonce={nonce}>
-    <header>
-      <StatusBanner licence={model} licenceType={licenceType} />
+    <Provider store={store}>
+      <header className="pdf-header">
+        {
+          licenceType === 'ppl'
+            ? <ProjectStatusBanner model={model} versionId={versionId} isPdf={true} />
+            : <LicenceStatusBanner licence={model} licenceType={licenceType} isPdf={true} />
+        }
 
-      <p className="float-left">OFFICIAL - SENSITIVE</p>
+        <p className="float-left">OFFICIAL - SENSITIVE</p>
 
-      <p className="float-right">
-        <Fragment>{licenceType.toUpperCase()} number: <strong>{model.licenceNumber}</strong></Fragment>
+        <p className="float-right">
+          { model.licenceNumber &&
+            <Fragment>{licenceType.toUpperCase()} number: <strong>{model.licenceNumber}</strong></Fragment>
+          }
 
-        <Fragment> | Granted: <strong>{format(model.issueDate)}</strong></Fragment>
+          { model.issueDate &&
+            <Fragment> | Granted: <strong>{format(model.issueDate)}</strong></Fragment>
+          }
 
-        { ['pil', 'pel'].includes(licenceType) && (moment(model.updatedAt).isAfter(model.issueDate, 'day')) &&
-          <Fragment> | Amended: <strong>{format(model.updatedAt)}</strong></Fragment>
-        }
+          { ['pil', 'pel'].includes(licenceType) && model.status === 'active' && (moment(model.updatedAt).isAfter(model.issueDate, 'day')) &&
+            <Fragment> | Amended: <strong>{format(model.updatedAt)}</strong></Fragment>
+          }
 
-        { licenceType === 'ppl' && model.amendedDate &&
-          <Fragment> | Amended: <strong>{format(model.amendedDate)}</strong></Fragment>
-        }
+          { licenceType === 'ppl' && model.amendedDate &&
+            <Fragment> | Amended: <strong>{format(model.amendedDate)}</strong></Fragment>
+          }
 
-        { licenceType === 'ppl' && model.status !== 'expired' && model.status !== 'revoked' &&
-          <Fragment> | Expires: <strong>{format(model.expiryDate)}</strong></Fragment>
-        }
+          { licenceType === 'ppl' && model.issueDate && model.status !== 'expired' && model.status !== 'revoked' &&
+            <Fragment> | Expires: <strong>{format(model.expiryDate)}</strong></Fragment>
+          }
 
-        { model.status === 'expired' &&
-          <Fragment> | Expired: <strong>{format(model.expiryDate)}</strong></Fragment>
-        }
+          { model.status === 'expired' &&
+            <Fragment> | Expired: <strong>{format(model.expiryDate)}</strong></Fragment>
+          }
 
-        { model.status === 'revoked' &&
-          <Fragment> | Revoked: <strong>{format(model.revocationDate)}</strong></Fragment>
-        }
-      </p>
-    </header>
+          { model.status === 'revoked' &&
+            <Fragment> | Revoked: <strong>{format(model.revocationDate)}</strong></Fragment>
+          }
+        </p>
+      </header>
+    </Provider>
```

Comment:
It's a bit out of scope here, but it seems like a lot of the things in this file are switching on and off depending on whether it's a PPL or not. There's probably a case for just splitting the whole thing into two components with minimal logic.